chime/edit_functions.py

```diff
@@ -68,17 +68,18 @@ def upload_new_file(clone, dir_path, upload):
 
     return file_path
 
-def delete_file(clone, path, file_name):
-    ''' Delete a file from the working directory, return its path.
+def delete_file(clone, file_path):
+    ''' Delete files from the working directory, return their paths.
     '''
-    file_path = join(path or '', file_name)
     full_path = join(clone.working_dir, file_path)
     do_save = False
 
-    if isdir(full_path):
-        rmdir(full_path)
-    elif exists(full_path):
-        remove(full_path)
-        do_save = True
+    removed_paths = []
+    if exists(full_path):
+        removed_path_notes = clone.git.rm('-r', full_path).splitlines()
+        # paths are returned by git rm in the format: "rm 'path/goes/here'"; extract the paths
```

Comment:
This is weird. I’m a uncomfortable with relying on git command line output; it changes between git versions in subtle ways I’m not confident that testing will uncover this reliably. Can you use `os.walk` instead to delete the contents after asking git to remove them recursively?